src/CommonStyles.js

```diff
@@ -18,7 +18,9 @@ export const labelCommon = css`
   border-radius: var(--http-method-label-border-radius, 3px);
   font-weight: var(--http-method-label-font-weigth, 400);
   font: inherit;
-  font-size: inherit;
+  font-size: var(--http-method-label-font-size, 14px);
```

Comment:
The default font size is a breaking change for this component. Both ARC and APIC uses font size 15px by default and ARC has configurable value in the application settings. This has to be inherited from the parent.